CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+- Fix RandomIdGenerator can generate invalid Span/Trace Ids
+  ([#3921](https://github.com/open-telemetry/opentelemetry-python/issues/3921))
```

Comment:
I think you should replace this with your PR id instead of issue id

Reply:
Yes, thanks!